x-pack/plugins/stack_connectors/public/connector_types/bedrock/bedrock.tsx

```diff
@@ -21,7 +21,7 @@ export function getConnectorType(): BedrockConnector {
     id: BEDROCK_CONNECTOR_ID,
     iconClass: lazy(() => import('./logo')),
     selectMessage: i18n.translate('xpack.stackConnectors.components.bedrock.selectMessageText', {
-      defaultMessage: 'Send a request to AWS Bedrock systems.',
+      defaultMessage: 'Send a request to AWS Bedrock.',
```

Comment:
[The commit got lost ](https://github.com/elastic/kibana/commit/6f9a23ffdd66724fbde5cde1bb14b898a150d95e)for these couple of translations from the renaming PR, so I am tucking them in here. hope that is ok